inlong-common/src/main/java/org/apache/inlong/common/msg/InLongMsg.java

```diff
@@ -246,13 +246,12 @@ public boolean addMsg(String attr, ByteBuffer data) {
             return false;
         }
 
-        DataBuffer outputBuffer = attr2MsgBuffer.get(attr);
-        if (outputBuffer == null) {
-            outputBuffer = new DataBuffer();
-            attr2MsgBuffer.put(attr, outputBuffer);
+        DataBuffer outputBuffer = attr2MsgBuffer.computeIfAbsent(attr, key -> {
+            DataBuffer newBuffer = new DataBuffer();
             // attrlen + utflen + meglen + compress
             this.datalen += attr.length() + 2 + 4 + 1;
-        }
+            return newBuffer;
+        });
```

Comment:
```java
DataBuffer outputBuffer = attr2MsgBuffer.computeIfAbsent(attr, key -> {
            DataBuffer newBuffer = new DataBuffer();
            // attrlen + utflen + meglen + compress
            this.datalen += attr.length() + 2 + 4 + 1;
            return newBuffer;
        });
```
computeIfAbsent should only create a new DataBuffer if the mapping for attr does not exist.
These two lines should move out of the mappingFunction.
```java
            // attrlen + utflen + meglen + compress
            this.datalen += attr.length() + 2 + 4 + 1;
```
Suggestion:
```java
        attr2MsgBuffer.computeIfAbsent(attr, k-> new DataBuffer());
         // attrlen + utflen + meglen + compress
        this.datalen += attr.length() + 2 + 4 + 1;
```